.storybook/preview.js

```diff
@@ -2,7 +2,7 @@
  * Preview.js is loaded in the Canvas tab, the “preview” iframe that renders your components in isolation. Use preview.js
  * for global code (such as CSS imports or JavaScript mocks) that applies to all stories.
  */
-import '../styles/globals.scss'
+import '../public/styles/globals.scss'
```

Comment:
@rocketnova I added this in a new commit, but I think its trivial enough that I'm approving & merging